### PR TITLE
Fix console error in `sticky-sidebar`

### PR DESCRIPTION
**Pull request by yakov116**
Fixes a console error in `sticky-sidebar`

Test: Go to conversation tab then go to the files tab. The resize observer was never disconnected so it will error out.

### CODE REVIEW

**Comment by fregante** [comment]
Is this executed in order if the sidebar is regenerated? I'd expect `remove` to be called before `add`, but I'm not sure. If it's the opposite, it won't work since `.disconnect()` will forget all the observed elements

**Comment by yakov116** [comment]
test comment

**Comment by yakov116** [comment]
Seems to be working for me.

File: source/features/sticky-sidebar.tsx [hunk 1]
```diff
@@ -16,11 +16,12 @@ function updateStickiness(): void {
 }
 
 const onResize = debounce(updateStickiness, {wait: 100});
+const observer = new ResizeObserver(onResize);
 
 function init(): void {
 	observe(sidebarSelector, {
 		add(sidebar) {
```
Comment (yakov116) [inline source/features/sticky-sidebar.tsx:23]: Why does this not have a "not selector"

Comment (fregante) [reply source/features/sticky-sidebar.tsx:23]: It’s not making any changes. We’re just listening to it. ResizeObserver already deduplicates identical .observe calls, I suppose

Comment (yakov116) [reply source/features/sticky-sidebar.tsx:23]: ok

**Comment by fregante** [comment]
I’m a bit lost on this feature at the moment but I trust you tested it so it can be merged

**Comment by fregante** [comment]
1. https://github.com/sindresorhus/refined-github
2. Visit Issues
3. Resize window

```
refined-github.js:10773 Uncaught TypeError: Cannot read property 'classList' of undefined
    at updateStickiness (refined-github.js:10773)
    at later (refined-github.js:197)
```

**Pull request merged**